MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include *.py
+include *.txt
```

Comment:
How about renaming CHANGES.txt to CHANGES.rst?

Reply:
I will do it in the further process on the road to Python 3.7, this PR is mainly for infrastructure.